Source/Evolve/WarpXEvolve.cpp

```diff
@@ -330,64 +336,61 @@ WarpX::OneStep_nosub (Real cur_time)
     if (do_pml && pml_has_particles) CopyJPML();
     if (do_pml && do_pml_j_damping) DampJPML();
 
```

Comment:
The diff below is confusing, but here I simply remove the `if` condition (`if( do_electrostatic == ElectrostaticSolverAlgo::None ) {`), since we do not call the function `OneStep_nosub` anymore, when running in electrostatic mode.